### PR TITLE
Introduce artman config for Ruby

### PR DESCRIPTION
**Pull request by jmuk**

### CODE REVIEW

**Comment by jmuk** [comment]
@geigerj FYI

File: gapic/api/artman_pubsub.yaml [hunk 1]
```diff
@@ -22,3 +22,5 @@ pubsub_csharp:
   final_repo_dir: ${REPOROOT}/artman/output/gcloud-csharp-pubsub
 pubsub_php:
   final_repo_dir: ${REPOROOT}/artman/output/gcloud-php-pubsub
+pubsub_ruby:
+  final_repo_dir: ${REPODIR}/artman/output/gcloud-ruby-pubsub
```
Comment (garrettjonesgoogle) [inline gapic/api/artman_pubsub.yaml]: don't you want REPOROOT here?

Comment (jmuk) [inline gapic/api/artman_pubsub.yaml]: Oops, thank you for catching this. This is a typo. fixed.

**Comment by garrettjonesgoogle** [comment]
LGTM

**Pull request merged**